types/har-format/index.d.ts

```diff
@@ -421,15 +455,15 @@ export interface Entry {
     /** _non-standard_  */
     _index?: number | null | undefined;
     /** _non-standard_  */
-    _initiator?: string | null | undefined;
+    _initiator?: Initiator | string | null | undefined;
     /** _non-standard_  */
-    _initiator_column?: string | null | undefined;
+    _initiator_column?: number | null | undefined;
```

Comment:
Are there use-cases when `_initiator_column` or `_initiator_line` could be of type `sting`?

Reply:
None that I know of. They are conceptually numeric and I've only seen them as numbers. Seems to have just been a mistake.